docs/faq.rst

```diff
@@ -252,23 +252,19 @@ not limited to):
 How can I use the Dockstore CLI with Python 2?
 ----------------------------------------------
 
-Python 2 support ends in 2020. You can get the Python 2 requirements.txt with ``curl -o requirements.txt "https://dockstore.org/api/metadata/runner_dependencies?python_version=2"``
+Python 2 support ended in 2020. You can get the Python 2 requirements.txt with ``curl -o requirements.txt "https://dockstore.org/api/metadata/runner_dependencies?python_version=2"``
 but it is currently untested.
 
-How do I add other users as maintainers of a tool?
---------------------------------------------------
+How do I add other users as maintainers of a workflow?
+------------------------------------------------------
 
-For tools registered on Quay.io and workflows registered with GitHub,
-Dockstore automatically allows users from the same Quay.io organization
-or GitHub organization to manage tools/workflows together (users will
-need to "Refresh Organization" or "Refresh All").
+For workflows registered with GitHub, Dockstore automatically allows users from 
```

Comment:
I suggest something like this, needs tweaking as I ran out of steam:


For workflows registered with GitHub, Dockstore allows users from the same GitHub organization to manage workflows together. If a new GitHub workflow is added to Dockstore by another user, you may need to tell Dockstore to add it to your workflows.

If the workflow was added to Dockstore via the GitHub App, click Discover Existing Dockstore Workflows.

If the workflow was added by manually registering it, click Refresh Organization.


@aofarrel Thoughts?

@denis-yuen @garyluu , in writing this suggestion, it seems like the web service could automatically do this -- when a workflow is added to Dockstore, the web service could find all the other GitHub users for that org and auto add them to the workflow. Is there a reason we don't do that already? If not, I'll create a new issue.

Reply:
We don't auto-add them to the organization because then the other users complain that their my-workflows and my-tools is bloated (and we're missing a search to filter those out)

Reply:
> If the workflow was added to Dockstore via the GitHub App, click Discover Existing Dockstore Workflows.
If the workflow was added by manually registering it, click Refresh Organization.

I don't think this is right...I think the Discover existing workflow is used when you don't see an entire organization listed and doesn't really have anything to do with the type of workflow it is.

Reply:
https://github.com/dockstore/dockstore-ui2/pull/984

Reply:
> 

Aargh, good catch, you're right. I'd seen [this code](https://github.com/dockstore/dockstore/blob/477977c5e657c3c2339f64da6e6f905ba5513aa5/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java#L943) and somehow convinced myself the Discover button was calling that.

Reply:
So my suggestion becomes something like:

For workflows registered with GitHub, Dockstore allows users from the same GitHub organization to manage workflows together. If a new GitHub workflow from the same GitHub organization is added to Dockstore by another user, click the Discover Existing Dockstore Workflows button so it will appear in My Workflows.

Reply:
@denis-yuen @garyluu  

I think we can consider auto-add and discuss  in a new ticket. Not sure avoiding this is the best way of avoiding bloat (wouldn't they get it eventually if they ever hit discover?)  A filter or "I am not interested" marker is reasonable too.  

That said, it doesn't avoid the need for a discover button or this FAQ since other events can occur that change access without a workflow being added to Dockstore. (Being added to an org, being given access to a repo, etc.)